stateoftheuniverse/widgets/moon_functions.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/python3
+"""
+A function to return moon phase & moon brightness.
+"""
+
+import numpy as np
```

Comment:
I personally like to sort my imports alphabetically (and group `import` and `from ... import ...` statements separately), but I guess this is just a matter of taste :)